src/main/kotlin/org/rust/ide/actions/RustGoToSuperModAction.kt

```diff
@@ -13,6 +14,11 @@ class RustGoToSuperModAction : DumbAwareAction() {
         val psiFile = e?.getRequiredData(CommonDataKeys.PSI_FILE) ?: return
         val superModFile = psiFile.rustMod?.`super`?.containingFile?.virtualFile ?: return
 
-        OpenFileDescriptor(project, superModFile, 0).navigate(true)
+        val descriptor = OpenFileDescriptor(project, superModFile, 0)
+
+        val targetEditor = FileEditorManager.getInstance(project).openTextEditor(descriptor, true);
+        if (targetEditor != null) {
+            targetEditor.caretModel.moveToOffset(0);
```

Comment:
Hm, `OpenFileDescriptor(project, superModFile, 0, 0)` should do the necessary movement?